herddb-core/src/main/java/herddb/index/brin/BlockRangeIndex.java

```diff
@@ -738,7 +805,9 @@ public void unload(long pageId) {
 
         @Override
         public String toString() {
-            return "Block{" + "key=" + key + ", minKey=" + key.minKey + ", size=" + size + ", next=" + (next == null ? null : next.key) + "}";
+            return System.identityHashCode(this) + "Block{" + "key=" + key + ", minKey=" + key.minKey
```

Comment:
is identityHashCode a left over ? 
 if you want to keep it we can put it inside the curly braces